src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.h

```diff
@@ -32,41 +33,49 @@ namespace ros
 
 namespace Util
 {
-    using LayerMsg    = thunderbots_msgs::DrawLayer;
-    using LayerMsgMap = std::map<std::string, LayerMsg>;
-    using ShapeMsg    = thunderbots_msgs::DrawShape;
     using time_point  = std::chrono::time_point<std::chrono::system_clock>;
 
     class VisualizerMessenger
     {
        public:
         /**
-         * DrawStyle is a struct that packs the appearance properties that goes into a
+         * ShapeStyle is a struct that packs the appearance properties that goes into a
          * shape message The default constructor contains the default value of the draw
          * style
          */
-        typedef struct DrawStyle
+        typedef struct ShapeStyle
         {
             // Struct constructor
-            DrawStyle() : fill("white"), stroke("black"), stroke_weight(1) {}
+            ShapeStyle() : texture(0), tint(0xFFFFFFF) {}
 
-            std::string fill;
-            std::string stroke;
-            uint8_t stroke_weight;
-        } DrawStyle;
+            uint8_t texture;
+            uint32_t tint;
+        } ShapeStyle;
 
-        /**
-         * DrawTransform is a struct that packs the transform properties that goes into a
-         * shape message The default constructor contains the default value of the
-         * transformation
-         */
-        typedef struct DrawTransform
+        typedef struct Shape
         {
-            DrawTransform() : rotation(0.0), scale(1.0) {}
-
-            double rotation;
-            double scale;
-        } DrawTransform;
+            Shape()
+                : layer(0),
+                  status(0),
+                  texture(0),
+                  x(0),
+                  y(0),
+                  width(1000),
+                  height(1000),
+                  rotation(0),
+                  tint(0xFFFFFFFF)
+            {
+            }
+            uint8_t layer;
+            uint8_t status;
+            uint8_t texture;
+            int16_t x;
+            int16_t y;
+            int16_t width;
+            int16_t height;
+            int16_t rotation;
+            uint32_t tint;
+        } Shape;
```

Comment:
nit, optional: in C++ you can write:
```
struct Shape {
...
};
```
and it should be equivalent.

Reply:
Can you expand on this? Do you mean to replace them with ellipses (`...`) and have the struct constructor implicitly make the members?

Reply:
Whoops sorry, I see how that could be ambiguous. I was being lazy and meant the `...` to indicate all the current struct members, ie:
```
struct Shape {
    uint8_t layer;
    uint8_t status;
    uint8_t texture;
    int16_t x;
    int16_t y;
    int16_t width;
    int16_t height;
    int16_t rotation;
    uint32_t tint;
};
```

Reply:
Done

Reply:
doesn't look like this is changed?

Reply:
Done